resources/views/admin/sleeps/create.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('layouts.default')
+
+@section('title')
+Add Sleep
+@parent
+@stop
+
+
+
+@section('content')
+<div class="content-wrapper">
+    <!-- Main content -->
+    <section class="content">
+        <div class="row">
+            <div class="col-lg-12 col-md-12">
+                <div class="box">
+                    <div class="box-header">
+                        <div class="row">
+                            <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
```

Comment:
col-6 will do the trick. Test it it well before finalizing. Also make same changes in other files

Reply:
Same as above